docs/compilers/CSharp/Compiler Breaking Changes - DotNet 5.md

```diff
@@ -58,3 +58,11 @@
             o is sbyte or short or int or long;
     ```
     Because the `and` and `or` combinators can follow a type pattern, the compiler interprets them as part of the pattern combinator rather than an identifier for the declaration pattern. Consequently, it is an error to use `or` or `and` as pattern variable identifiers starting with C# 9.0.
+
+4. The fix for https://github.com/dotnet/roslyn/issues/44067 generates correct (different) code.
+   In certain cases the compiler used to generate code whose behavior was ambiguous according
+   to the CLR's specification. The compiler used to produce the warning CS1957 in those cases.
```

Comment:
>The compiler used to produce the warning CS1957 in those cases. [](start = 31, length = 63)

This statement probably should be adjusted once the change around warming reporting is reverted.